### PR TITLE
Fix getApplicationDefault to also return projectId.

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

**Comment by jmdobry** [comment]
@murgatroid99 PTAL

**Comment by jmdobry** [comment]
ping? @murgatroid99 @bjwatson

**Comment by bjwatson** [comment]
I'm not familiar enough with NodeJS to review this. @jmuk do you have a couple minutes to take a look?

File: lib/auth/googleauth.js [hunk 1]
```diff
@@ -37,9 +37,9 @@ function GoogleAuth() {
 }
 
 // Executes the given callback if it is not null.
-function callback(c, err, res) {
+function callback(c, err, arg1, arg2) {
   if (c) {
-    c(err, res);
+    c(err, arg1, arg2);
```
Comment (jmuk) [inline lib/auth/googleauth.js]: +1, although I'm not seeing usage of >2 args right now.

Comment (PlasmaPower) [inline lib/auth/googleauth.js]: Yeah, it's just future proofing of course.

Comment (jmdobry) [inline lib/auth/googleauth.js]: Are we sure we want to do this? Won't it wipeout the user's context if they bound their callback to something? e.g.

```js
function callback(c) {
  if (c) {
    c.apply(null, Array.prototype.slice.call(arguments, 1));
  }
}
```

Comment (PlasmaPower) [inline lib/auth/googleauth.js]: No, a bind will still work even if apply is called on the result. Think of a this-only bind like this:

```js
function bind(fn, self) {
  return function() {
    return fn.call(self, arguments);
  }
}
```

Example of this working:


```js
var fn = (function () { return this.a }).bind({a: "hello"});
console.log(fn.call(null, []));
```

(outputs hello)

Comment (jmdobry) [inline lib/auth/googleauth.js]: Did you mean to use `apply` in your examples?

Comment (jmdobry) [inline lib/auth/googleauth.js]: Fixed

**Comment by bjwatson** [comment]
No I can't. I'm not a member either.

@jmdobry can you make both @jmuk and I members of this repo? @jmuk should probably also be an owner.

**Comment by jmdobry** [comment]
I can't, as I am not an admin, though @justinbeckwith and I have wanted to be for a while.

**Comment by bjwatson** [comment]
@murgatroid99 @ofrobots, are you admins?

**Comment by stephenplusplus** [comment]
Looks like I can merge. Should I try to release also? :)

**Pull request merged**